docs/developer-lightspeed-guide/master-docinfo.xml

```diff
@@ -0,0 +1,11 @@
+<title>MTA Developer Lightspeed Guide</title>
+<productname>{DocInfoProductName}</productname>
+<productnumber>{DocInfoProductNumber}</productnumber>
+<subtitle>Using the {ProductName} command-line interface to migrate your applications</subtitle>
+<abstract>
+    <para>Use {ProductFullName} Developer Lightspeed for application modernization in your organization by running Artificial Intelligence-driven static code analysis for Java applications.</para>
```

Comment:
> running Artificial Intelligence-driven static code analysis for Java applications.

Note our analysis is not AI-driven, our code suggestions/fixes are AI-driven.  Our analysis is rules based static code analysis.